Sources/HMAC.swift

```diff
@@ -35,9 +35,12 @@ open class HMAC {
 
         let ipadAndMessageHash = SHA1(ipad + message).calculate()
         let mac = SHA1(opad + ipadAndMessageHash).calculate()
-
-        return Data(bytes: UnsafePointer<UInt8>(mac), count: mac.count)
-
+        var hashedData: Data? = nil
```

Comment:
this is art, transform one line code into closure...
`Data(withUnsafeBufferPointer: mac)` that done the same thing could be writted... (but yes maybe we can to do something else...